automated/linux/factory-reset/prepare-reset.sh

```diff
@@ -11,11 +11,14 @@ TYPE="factory_reset"
 ADDITIONAL_TYPE=""
 LABEL=""
 SOTA_CONFDIR="/etc/sota/conf.d"
+HSM_MODULE=""
 
 usage() {
     echo "\
     Usage: $0 [-t <factory_reset|factory_reset_keep_sota|factory_reset_keep_sota_docker>]
               [-a <factory_reset|factory_reset_keep_sota|factory_reset_keep_sota_docker>]
+              [-l <target label>]
+              [-s <hsm module>]
```

Comment:
Would it make sense to set this flag as '-S', since its that in automated/linux/ota-update/download-update.sh ?

Reply:
Good idea. Done

Reply:
Think you missed to change it from '[-s <hsm module>]' to '[-S <hsm module>]' here.